backend/src/kitconcept/intranet/controlpanels/intranet.py

```diff
@@ -83,3 +101,10 @@ def __call__(self, data):
         )
         data["kitconcept.intranet.custom_css"] = settings.custom_css
         data["kitconcept.person_squared_images"] = settings.person_squared_images
+        data["kitconcept.intranet.default_feedback_email"] = (
+            settings.default_feedback_email
+        )
+        data["kitconcept.intranet.feedback_cc_email"] = settings.feedback_cc_email
```

Comment:
```suggestion
```

These are not used on the client side and we should not expose them to normal editors. So, don't include them in the site endpoint here.